docs/webhooks/01-Overview.md

```diff
@@ -468,19 +465,14 @@ Depending on the `event.event_type`, of the webhook payload, the `event.data` fi
 &nbsp;
 ## Deprecated Versions
 &nbsp;
-### [V2 Webhooks](../webhooks/09-V2-Overview.md) Reaching End Of Support
-* V2 will be reaching End-Of-Support (EOS) by October 2022.<br>
-* V2 will be reaching End-Of-Life (EOL) by March 2023.<br>
-
-
 ### [V1 Webhooks](../webhooks/10-V1-Overview.md) Reached End Of Support
 * V1 has already reached End-Of-Support (EOS) in November 2021.<br>
 * V1 will be reaching End-Of-Life (EOL) by October 2022.<br>
 
 <!-- theme: info -->
 > ### What is EOS & EOL?
 > EOS means PagerDuty will not support any additional bug fixes or entertain new feature requests.<br>
-> E.g. After October 2022, App integrations built on V2 Webhooks will still continue to work, but any feature requests on V2 will not be entertained.
+> E.g. After November 2021, App integrations built on V2 Webhooks will still continue to work, but any feature requests on V1 will not be entertained.
```

Comment:
E.g. After November 2021, App integrations built on V1 will still work after November 2021 util the EOL date is reached, but any feature requests on V1 will not be entertained.